devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusGenerateCode.java

```diff
@@ -41,6 +43,16 @@ public QuarkusGenerateCode() {
         super("Performs Quarkus pre-build preparations, such as sources generation");
     }
 
+    /**
+     * Create a dependency on classpath resolution. This makes sure included build are build this task runs.
+     *
+     * @return resolved compile classpath
+     */
+    @CompileClasspath
+    public FileCollection getClasspath() {
+        return QuarkusGradleUtils.getSourceSet(getProject(), SourceSet.MAIN_SOURCE_SET_NAME).getCompileClasspath();
```

Comment:
Just curious, do you know what it was before this change?

Reply:
Actually, it was the same. This method declare a dependency on the compile classpath which will make sure that gradle resolve all dependency even included build (which was not always guaranteed). This has no effect on the compile classpath of the task.